### ISSUE TITLE
all: timeouts on `linux-s390x-ibm-race`

### ISSUE THREAD

**Issue by bcmills** [golang/go#60109]
```
#!watchflakes
post <- builder == "linux-s390x-ibm-race" && (`test timed out` || `failed to start: context deadline exceeded` || `WaitDelay expired before I/O complete`)
```

(attn @golang/s390x)

**Comment by srinivas-pokala** [comment]
> @srinivas-pokala, this builder probably needs a `GO_TEST_TIMEOUT_SCALE`.

yes @bcmills,  I am adding GO_TEST_TIMEOUT_SCALE to the build configuration

**Comment by srinivas-pokala** [comment]
Hi @bcmills  for this race builder I have added timeout GO_TEST_TIMEOUT_SCALE with this CL [(dashboard: set GO_TEST_TIMEOUT_SCALE for linux-s390x-ibm-race builder)](https://go-review.googlesource.com/c/build/+/495655) . Is this scaled value of GO_TEST_TIMEOUT_SCALE not optimum interms of test's some of which failing?.

**Comment by bcmills** [comment]
Hmm, but it only takes ~86s in most runs. Maybe this is on a release branch?

**Comment by bcmills** [comment]
Nope, those aren't on a release branch. I guess the test just takes way longer sometimes.

Maybe it's a VM isolation problem? (Is it possible that that test is ending up a lot slower on some runs due to contention of some sort with another VM, or another test on the same VM?)

**Comment by bcmills** [comment]
@golang/s390x, if this builder needs a longer timeout scale, please send a CL to update the builder configuration.

**Comment by srinivas-pokala** [comment]
Bumped up the builder resource,  there are no longer any timeouts.
